src/rantanplan/core.py

```diff
@@ -38,9 +54,1266 @@
 LIAISON_FIRST_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚy")
 LIAISON_SECOND_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚhy")
 STRESSED_UNACCENTED_MONOSYLLABLES = {"yo", "vio", "dio", "fe", "sol", "ti", "un"}
-UNSTRESSED_UNACCENTED_MONOSYLLABLES = {"me", "nos", "te", "os", "lo", "la", "los", "las", "le",
-                                       "les", "se", "tan", "el", "mas", "te", "si", "tu", "de",
-                                       "mi", "si", "tu", "que", "de", "su"}
+UNSTRESSED_UNACCENTED_MONOSYLLABLES = {'de', 'el', 'la', 'las', 'le', 'les', 'lo', 'los',
+                                       'mas', 'me', 'mi', 'nos', 'os', 'que', 'se', 'si',
+                                       'su', 'tan', 'te', 'tu'}
+
+
+"""
+Regular expressions and rules for syllabification exceptions
+
+"""
+
+
+"""
+Fonetic exceptions dict for potential diaeresis
+
+paranoia': (['pa', 'ra', 'noia'],
```

Comment:
Add comment to indicate this is the default syllabification.

Reply:
Done:
`paranoia': (['pa', 'ra', 'noia'], # Normative syllabification
         [
         (['pa', 'ra', 'no', 'ia'], (2, 3)), # Items to mark as a metaplasm, if they are used
         (['pa', 'ra', 'no', 'i', 'a'], (2, 3, 4))  # Another, fake, items to mark as a metaplasm
         ]),`